accepted/2025/cli-acquisition-tool.md

```diff
@@ -0,0 +1,217 @@
+# .NET SDK Acquisition and Management Tool
+
+**Owner** [Chet Husk](https://github.com/baronfel) | [Daniel Plaisted](https://github.com/dsplaisted)
+
+Users of the .NET SDK today face a wide array of choices when getting our tooling, with multiple distribution channels that release at different cadences.
+While these options serve specific needs, several important user cohorts are not well served and have asked for a more unified approach to getting started with our tooling.
+
+
+This proposal introduces a standalone install and SDK management tool (tentatively named `dnup`) that provides a unified interface for acquiring and managing .NET tooling across all supported platforms, focusing on user-local installations that can be managed (largely) without elevation.
+
+This tool is especially critical for users that are
+
+* new users, especially those from the Node or Rust ecosystems
+* Linux users who typically use packages from their distro, as not all distro maintainers support feature bands beyond 1xx
+* macOS users, who can access all feature bands but have no way of cleaning up old installations (ignoring the dotnet-core-uninstall tool)
+* users who manually perform _any_ SDK installations and therefore are responsible for clean up
+
+## Scenarios and User Experience
+
+### Brand-new learner
+
+Sasha wants to learn C# and develop applications on her Mac.
+She searches and is led to a 'getting started' style page that directs her to download our bootstrapper.
+She navigates to `get.dot.net` and clicks the prominent link for `dnup-init`.
+She then clicks the downloaded executable, which runs the bootstrap process where she is prompted through an interactive setup:
+
+```bash
+Where should we install .NET to? *(~/Library/Application Support/dnup)*
+> <enter>
+What version should we install? *(latest)*
+> <enter>
+Should we set up your environment/PATH automatically? *(Y/n)*
+> <enter>
+Installing .NET SDK 10.0.100 to ~/Library/Application Support/dnup
```

Comment:
I think the installation folder should be named `dotnet`.

```suggestion
Where should we install .NET to? *(~/Library/Application Support/dotnet)*
> <enter>
What version should we install? *(latest)*
> <enter>
Should we set up your environment/PATH automatically? *(Y/n)*
> <enter>
Installing .NET SDK 10.0.100 to ~/Library/Application Support/dotnet
```

Reply:
Fair - I do think the 'state' directory for this should be in a location that's more clearly-associated to the `dnup` tool though.